nh_eobs/models/nh_clinical_environment.py

```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8 -*-
+"""
+A module that provides information about the environment that eObs is running
+in.
+
+The module was created to fulfil the need to access server environment
+variables from the Javascript (e.g. getting eObs version to display in the
+'about' menu item on desktop).
+
+From Javascript there are 2 options for server communication:
+    * Call model methods.
+    * Call an RPC or regular HTTP endpoint.
+
+Calling a model seemed the better option as backend code could then also make
+use of it without having to unnecessarily call an endpoint.
+"""
+import logging
+from os import environ
+
+from openerp.models import AbstractModel
+
+
+_logger = logging.getLogger(__name__)
+
+
+class NhClinicalEnvironment(AbstractModel):
+    _name = 'nh.clinical.environment'
+
+    @staticmethod
+    def get_eobs_version(*args):
```

Comment:
Instead of using `try, catch` maybe use `environ.get('EOBS_VERSION')` and then do the log via `if not eobs_version` and then you can just return `eobs_version` as you're returning `None` at the end of the catch anyways